src/themes/brutalist.css

```
@@ -0,0 +1,832 @@
+  }
+
+  .product-card wa-button::part(prefix) {
+    padding-inline-start: var(--wa-space-xs);
+  }
+
+  .product-card wa-button::part(suffix) {
+    padding-inline-end: var(--wa-space-xs);
+  }
+
+  .product-card wa-button:not(:last-child) {
+    margin-inline-end: var(--wa-space-xs);
+  }
+
+  /* imitating the rounded avatar */
+  .blog wa-avatar::part(base) {
+    background-color: transparent;
+  }
+
+  .blog wa-avatar::part(image) {
+    border: var(--wa-border-width-m) solid var(--wa-color-brand-spot);
+    border-radius: var(--wa-corners-circle);
+  }
+
+  .blog .landscape-frame {
+    box-shadow: var(--wa-shadow-level-1);
+    border: var(--wa-panel-border-width) var(--wa-panel-border-style) var(--wa-color-surface-border);
+    margin-bottom: var(--wa-shadow-offset-y-level-1);
+  }
+
+  .post-meta .categories {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .message-composer wa-card::part(header) {
+    background-color: var(--wa-color-neutral-spot);
+  }
+
+  .message-composer .grouped-buttons {
+    --wa-color-neutral-border-subtle: color-mix(in oklab, var(--wa-color-base-30), white 40%);
+  }
+
+  .message-composer [slot='header'] wa-icon-button::part(base) {
+    color: var(--wa-color-neutral-text-on-spot);
+  }
+
+  .message-composer .grouped-buttons wa-icon-button::part(base) {
+    block-size: var(--wa-form-controls-height-s);
+    inline-size: var(--wa-form-controls-height-s);
+    justify-content: center;
+  }
+
+  .message-composer .grouped-buttons wa-icon-button::part(base):hover {
+    background-color: var(--wa-color-neutral-fill-highlight);
+    color: var(--wa-color-text-normal);
+  }
+
+  .message-composer wa-card::part(footer) {
+    border-width: 0;
+    --padding: 0 var(--wa-space-s) var(--wa-space-s) var(--wa-space-s);
+  }
+
+  .support-table th {
+    background-color: var(--wa-color-neutral-spot);
+    color: var(--wa-color-neutral-text-on-spot);
+  }
+
+  .support-table th wa-checkbox {
+    --background: transparent;
+    --border-color: var(--wa-color-neutral-text-on-spot);
+    --border-color-checked: var(--wa-color-neutral-text-on-spot);
+  }
+
+  .support-table tr {
+    border-width: var(--wa-border-width-m);
+  }
+
+  .checkout-form {
+    font-size: var(--wa-font-size-s);
+  }
+
+  .checkout-form .square-frame {
+    box-shadow: var(--wa-shadow-level-1);
+    border: var(--wa-panel-border-width) var(--wa-panel-border-style) var(--wa-color-surface-border);
+  }
+
+  .checkout-form wa-input::part(form-control-label),
+  .checkout-form wa-select::part(form-control-label) {
+    font-size: var(--wa-font-size-s);
+    font-weight: var(--wa-font-weight-medium);
+  }
+}
+
+.wa-theme-brutalist-dark,
+.wa-theme-brutalist-dark :host {
+  color-scheme: dark;
+
+  /**
+   * Base theme colors
+   */
+  --wa-color-surface-raised: var(--wa-color-base-10);
+  --wa-color-surface-default: var(--wa-color-base-20);
+  --wa-color-surface-lowered: black;
```

```suggestion
  --wa-color-surface-lowered: var(--wa-color-base-10);
```

This custom property is specifically meant for use cases like the `.preview-container` backdrop, so we could set this to our desired color for the container.